### PR TITLE
229 test data

### PR DESCRIPTION
**Pull request by danieldunderfelt**
UI PR for task https://github.com/HSLdevcom/bultti/issues/229

Server PR: https://github.com/HSLdevcom/bultti/pull/267

- Admin page for controlling test data
- Schema updates

### CODE REVIEW

**Review by Divinare — CHANGES_REQUESTED** [review]
* There was "WIP: hide test data for normal users" - is it done?
* Filter out "Testiliikennöitsijä A"

I would hope generated test data would have:
* 10 000m metriosuus per vehicle
* ages: vehicle ages: 1v, 1v, 10v, 10v
* why is there 3 vehicles in "Kaavion ajoneuvot päivätyypeittäin" - shouldn't there be 4?
* why is "Kaavion ajoneuvot kalustotyypeittäin" table empty?

testData_preInspection

File: src/page/AdminPage.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  useEffect(() => {
+    if (!isAdmin) {
+      window.history.back()
+    }
+  }, [isAdmin])
+
+  let [removeInspectionId, setRemoveInspectionId] = useState('')
+
+  let { data: inspection } = useQueryData(inspectionQuery, {
+    skip: removeInspectionId.length < 36,
```
[the remark this answers is not in the document]
Comment (danieldunderfelt) [reply src/page/AdminPage.tsx]: The length of a valid UUID. The idea is to not fetch the inspection preview with an incomplete UUID. I'll add a comment!

File: src/page/AdminPage.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    nextFetchPolicy: !removeInspectionId ? 'network-only' : 'cache-first',
```
[the remark this answers is not in the document]
Comment (danieldunderfelt) [reply src/page/AdminPage.tsx]: I don't think this matters at all in any way.

**Review by Divinare — APPROVED** [review]
LGTM!

**Pull request merged**